### ISSUE TITLE
sitemap ideation

### ISSUE THREAD

**Issue by pdehaan** [pdehaan/11ty-blog-ideas#17]
WIP

- /index.html
- /eleventy/index.html
- /eleventy/liquid/filters/abs/index.html
- /eleventy/liquid/filters/custom/{{nunjucks}}/index.html _(specific `.addLiquidFilter()` for filters cribbed from Nunjucks)_
- /eleventy/filters/custom/reading-time.html _(generic, global `.addFilter()`)_
- /eleventy/nunjucks/filters/wordcount/index.html
- /eleventy/config/dir/index.html
- /eleventy/config/data-deep-merge/index.html
- /puppeteer/devices/index.html
- /this-week-in-eleventy/2020-02-29/index.html

Not sure about convoluted tags like:
- "eleventy"
- "eleventy:liquid"
- "eleventy:liquid:filters"
- "eleventy:filters"
- "eleventy:config"
- "author:pdehaan"

Basically the tags would almost mirror the directory structure and have directory level data files and `.setDataDeepMerge()` so templates only need to specify example specific tags. Then, archive pages can look for an auto-collection awkwardly named: `collections["eleventy:liquid:filters"]`, or wherever the archive page lives in the tree.

But essentially the directory structure is very loose and unpredictable and fluid.

**Comment by pdehaan** [comment]
each example/post to have a _near_ required "repo" key in front-matter which points to a GitHub repo with working example. This would really suck for updating major 11ty versions though. 🤷‍♂

**Comment by pdehaan** [comment]
revisit idea of using my main repo:
- github.com/pdehaan/pdehaan.dev/
- github.com/pdehaan/11ty-foo-bar/
- github.com/pdehaan/random-unrelated-repos
 
vs custom org just for site+examples:
- github.com/pdehaan.dev/pdehaan.dev/
- github.com/pdehaan.dev/.ideas/
- github.com/pdehaan.dev/11ty-foo-bar/
